README.md

```diff
@@ -1,8 +1,38 @@
-[![Build Status](CI_BADGE_URL goes here)](REPO_URL goes here)
+[![Build Status](https://github.com/blyscuit/docs/actions/workflows/deploy_fly.yml/badge.svg)](https://github.com/blyscuit/rails-ic/tree/chore/3-deployment-automation)
 
 ## Introduction
 
-> *App introduction goes here ...*
+Rails Internal Certificate.
+
+
+## Get Started
```

Comment:
As the getting started is added here, should we remove the "Getting started" file from the wiki (.github/wiki/Getting-Started.md) and from the menu of the wiki?

Reply:
I will rename this one to `Quick Start`, I would like to keep ``wiki/get-started` for more details instructions. 🙏